
{'date': 'Wed Mar 24 2021 17:57:08.697 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 27
    main()
  module <module> line 25
    Associa(j = Jogo())
  module <module> line 21
    self.mito = self.Nome(self.cena)
  module <module> line 14
    self.nome = J.a(self.CELULA, x=400, y=50, w=100, h=50, cena=cena)
AttributeError: 'Nome' object has no attribute 'CELULA'
'''},
{'date': 'Wed Mar 24 2021 18:04:27.330 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  self.nome = J.a(Associa.VAZIO, x=400, y=50, w=90, h=30, o=0.9, style={"font-size":"20px"} cena=cena)
                                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 24 2021 18:08:36.892 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  self.mito = self.Nome(nome="mitocôndria"x=600, y=250, cena=self.cena)
                                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 24 2021 19:24:43.875 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: undefined has no properties
  module '$exec_2652' line 30
        def __init__(self, nome, tit, x, y, cena, lacuna=Associa.VAZIO, legenda=Associa.VAZIO):
'''},
{'date': 'Wed Mar 24 2021 19:25:39.343 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  VAZIO = "https://i.imgur.com/npb9Oej.png"
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Mar 24 2021 19:37:30.751 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 70
  if self.pontua = 4:
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 24 2021 19:42:49.649 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 73
  J.t( cena=self.cena, "Você acertou tudo! Parabéns!").vai()
                                                     ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Thu Mar 25 2021 12:05:03.566 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 128
    main()
  module <module> line 126
    Swap(j=J, img=Associa.CELULA, cena=J.c(self.CENA), w=700,h=600,x=100,y=50,dw=3,dh=3)
AttributeError: 'GUI' object has no attribute 'CENA'
'''},
{'date': 'Thu Mar 25 2021 12:05:35.200 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 128
    main()
  module <module> line 126
    Swap(j=J, img=Associa.CELULA, cena=J.c(Associa.CENA), w=700,h=600,x=100,y=50,dw=3,dh=3)
  module <module> line 113
    self.correto = " ".join(pecas)
TypeError: sequence item 0: expected str instance, int found
'''},
{'date': 'Thu Mar 25 2021 16:46:26.119 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 124
  self.venceu = J.n(cena, "Voce venceu!)
                                        ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Mar 29 2021 11:04:37.510 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: obj is undefined
  module '$exec_320' line 44
        def __init__(self, x, y, cena, jogo, img=Droner.KNOB):
'''},
{'date': 'Mon Mar 29 2021 11:05:30.124 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 67
    main()
  module <module> line 64
    Droner(cena)
  module <module> line 58
    self.Anteparo(100, 100, cena, self)
  module <module> line 46
    super().__init__(img, x=lx, y=ly, w=pw, h=ph, cena=cena)
NameError: name 'Anteparo' is not defined
'''},
{'date': 'Mon Mar 29 2021 11:05:44.659 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 67
    main()
  module <module> line 64
    Droner(cena)
  module <module> line 58
    self.Anteparo(100, 100, cena, self)
  module <module> line 46
    super().__init__(img, x=x, y=y, w=pw, h=ph, cena=cena)
NameError: name 'Anteparo' is not defined
'''},
{'date': 'Mon Mar 29 2021 11:08:21.567 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 67
    main()
  module <module> line 64
    Droner(cena)
  module <module> line 58
    Droner.Anteparo(100, 100, cena, self)
  module <module> line 46
    super().__init__(img, x=x, y=y, w=pw, h=ph, cena=cena)
NameError: name 'Anteparo' is not defined
'''},
{'date': 'Mon Mar 29 2021 11:17:32.412 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 90
    main()
  module <module> line 87
    Droner(cena)
  module <module> line 56
    class Drone(J.a):
  module <module> line 67
    def __init__(self, x, y, cena, jogo, img=Self.DRONE):
NameError: name 'Self' is not defined
'''},
{'date': 'Mon Mar 29 2021 11:42:55.508 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  CIS = {ROSA.norte: Rosa.leste, ROSA.leste: Rosa.norte, ROSA.sul: ROSA.oeste, ROSA.oeste, ROSA.sul}
                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 29 2021 11:43:45.557 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  CIS = {ROSA.norte: ROSA.leste, ROSA.leste: ROSA.norte, ROSA.sul: ROSA.oeste, ROSA.oeste, ROSA.sul}
                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 29 2021 11:49:55.752 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  CIS = {ROSA.norte: ROSA.leste, ROSA.leste: ROSA.norte, ROSA.sul: ROSA.oeste, ROSA.oeste, ROSA.sul}
                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 29 2021 11:50:47.265 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from collections import nametuple
ImportError: cannot import name 'nametuple'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    from collections import nametuple
'''},
{'date': 'Mon Mar 29 2021 11:51:10.409 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 17
    Rosa = nametuple("Rosa", "norte leste sul oeste")
NameError: name 'nametuple' is not defined
'''},
{'date': 'Mon Mar 29 2021 11:54:18.712 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 105
    main()
  module <module> line 102
    Droner(cena)
  module <module> line 93
    set_timeout(self.inicia, "1000")
AttributeError: 'Droner' object has no attribute 'inicia'
'''},
{'date': 'Mon Mar 29 2021 12:41:52.785 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 108
  def rodar(self, , rodado=0):
                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 29 2021 14:01:26.146 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 140
    main()
  module <module> line 137
    Droner(cena)
  module <module> line 115
    self.anteparos = [self.cria(index) for index in range(w*6)]
NameError: name 'w' is not defined
'''},
{'date': 'Mon Mar 29 2021 14:02:43.198 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 141
    main()
  module <module> line 138
    Droner(cena)
  module <module> line 116
    self.anteparos = [self.cria(index) for index in range(self.w*6)]
  module <module> line 124
    return Anteparo(x, y, cena, self) if good else Borda(x, y, cena, self) 
  module <module> line 72
    super().__init__(x=x, y=y, jogo=jogo, cena=cena, img=BORDA)
NameError: name 'BORDA' is not defined
'''},
{'date': 'Mon Mar 29 2021 16:11:56.903 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 126
  self.drone = Drone(index = self.w+1, cena, self)
                                           ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Mon Mar 29 2021 16:12:29.547 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $locals.index is undefined
  module '$exec_2074' line 155
main()
'''},
{'date': 'Mon Mar 29 2021 16:14:11.258 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 155
    main()
  module <module> line 152
    Droner(cena)
  module <module> line 126
    self.drone = Drone(self.w+1, cena, self)
  module <module> line 97
    self.x, self.y = self.jogo.localiza(index)
  module _spy.vitollino.main line 684
    self.elt.style.left = "{}px".format(value)
AttributeError: 'Drone' object has no attribute 'elt'
'''},
{'date': 'Mon Mar 29 2021 16:43:18.924 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    main()
  module <module> line 160
    Droner(cena)
  module <module> line 134
    self.drone = Drone(self.w+1, cena, self)
  module <module> line 105
    x, y, _ = [coor + GAP//4 for coor in self.jogo.localiza(index)]
TypeError: unsupported operand types for +: 'NoneType' and 'int'
'''},
{'date': 'Mon Mar 29 2021 16:44:31.643 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: this.tree[2] is undefined>
'''},
{'date': 'Mon Mar 29 2021 16:46:05.547 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    main()
  module <module> line 160
    Droner(cena)
  module <module> line 134
    self.drone = Drone(self.w, cena, self)
  module <module> line 105
    x, y, _ = [(coor + GAP//4) if coor else None for coor in self.jogo.localiza(index)]
  module <module> line 149
    return self.anteparos[index].localiza()
  module <module> line 85
    from random import choose, randint
ImportError: cannot import name 'choose'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    main()
  module <module> line 160
    Droner(cena)
  module <module> line 134
    self.drone = Drone(self.w, cena, self)
  module <module> line 105
    x, y, _ = [(coor + GAP//4) if coor else None for coor in self.jogo.localiza(index)]
  module <module> line 149
    return self.anteparos[index].localiza()
  module <module> line 85
    from random import choose, randint
'''},
{'date': 'Mon Mar 29 2021 16:46:33.874 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    main()
  module <module> line 160
    Droner(cena)
  module <module> line 134
    self.drone = Drone(self.w, cena, self)
  module <module> line 105
    x, y, _ = [(coor + GAP//4) if coor else None for coor in self.jogo.localiza(index)]
TypeError: can only concatenate list (not "int") to list
'''},
{'date': 'Tue Mar 30 2021 12:40:22.992 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 177
    main()
  module <module> line 174
    Droner(cena)
  module <module> line 147
    self.drone = Drone(int(1.25*GAP), int(1.75*GAP), cena, self)
  module <module> line 116
    x, y, _ = self.jogo.localiza(index)
AttributeError: 'Cena' object has no attribute 'localiza'
'''},
{'date': 'Tue Mar 30 2021 13:23:52.605 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 155
    main()
  module <module> line 152
    Droner(cena)
  module <module> line 125
    self.drone = Drone(self.w, cena, self)
  module <module> line 114
    img=self.DRONE
AttributeError: 'Drone' object has no attribute 'DRONE'
'''},
{'date': 'Wed Mar 31 2021 14:25:41.826 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 189
    main()
  module <module> line 186
    Droner(cena)
  module <module> line 157
    self.a = A(0, 0, cena, self)
TypeError: __init__() takes 1 positional argument but more were given
'''},
{'date': 'Wed Mar 31 2021 14:32:07.280 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 47
  def __init__(self, , index, cena, jogo, img=self.DRONE):
                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 31 2021 14:58:17.991 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    main()
  module <module> line 160
    associa = Associa()
  module <module> line 66
    self.mito = self.Nome(nome="mitocôndria", tit=0, x=650, y=150, jogo=self, cena=self.cena)
AttributeError: 'Associa' object has no attribute 'cena'
'''},
{'date': 'Wed Mar 31 2021 15:01:48.917 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 72
  def nome(nome, tit=0, x=650, y=150, jogo=self, cena=self.cena)
                                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 31 2021 15:49:27.186 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 171
    main()
  module <module> line 164
    associa = Associa(cena)
TypeError: __init__() missing 1 positional argument: caixa
'''},
{'date': 'Mon Apr 05 2021 15:38:34.864 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 195
    main()
  module <module> line 192
    Droner(cena)
  module <module> line 163
    self.a = Drone(22, cena, self)
NameError: name 'Drone' is not defined
'''},
{'date': 'Mon Apr 05 2021 15:44:00.490 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 196
    main()
  module <module> line 193
    Droner(cena)
  module <module> line 164
    self.a = Drone(22, cena, self)
  module <module> line 135
    x, y, _ = [(coor + GAP//4)  for coor in (x, y)]
ValueError: need more than 2 values to unpack
'''},
{'date': 'Mon Apr 05 2021 19:04:06.124 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 199
    main()
  module <module> line 196
    Droner(cena)
  module <module> line 168
    self.drone = Drone(0, cena, self)
  module <module> line 130
    x, y, azimuth = self.jogo.localiza(index)
ValueError: too many values to unpack (expected 3)
'''},
{'date': 'Mon Apr 05 2021 19:33:53.423 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 204
    main()
  module <module> line 201
    Droner(cena)
  module <module> line 169
    self.drone = Drone(0, cena, self)
  module <module> line 131
    self.index, x, y, azimuth = self.jogo.localiza(index)
  module <module> line 190
    return self.anteparos[index].localiza()
  module <module> line 113
    self.jogo.start()
  module <module> line 175
    self.drone.o = 0.5
AttributeError: 'Droner' object has no attribute 'drone'
'''},
{'date': 'Mon Apr 05 2021 19:42:28.181 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 204
    main()
  module <module> line 201
    Droner(cena)
  module <module> line 169
    self.drone = Drone(0, cena, self)
  module <module> line 131
    self.index, x, y, azimuth = self.jogo.localiza(index)
  module <module> line 190
    return self.anteparos[index].localiza()
  module <module> line 113
    self.jogo.start()
  module <module> line 175
    self.drone.o = 0.5
AttributeError: 'Droner' object has no attribute 'drone'
'''},
{'date': 'Mon Apr 05 2021 19:43:16.644 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 204
    main()
  module <module> line 201
    Droner(cena)
  module <module> line 169
    self.drone = Drone(0, cena, self)
  module <module> line 131
    self.index, x, y, azimuth = self.jogo.localiza(index)
  module <module> line 190
    return self.anteparos[index].localiza()
  module <module> line 113
    self.jogo.start()
  module <module> line 176
    self.drone.o = 0.5
AttributeError: 'Droner' object has no attribute 'drone'
'''},
{'date': 'Mon Apr 05 2021 19:43:49.403 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 204
    main()
  module <module> line 201
    Droner(cena)
  module <module> line 169
    self.drone = Drone(0, cena, self)
  module <module> line 131
    self.index, x, y, azimuth = self.jogo.localiza(index)
  module <module> line 190
    return self.anteparos[index].localiza()
  module <module> line 113
    self.jogo.start()
  module <module> line 176
    x = self.drone
AttributeError: 'Droner' object has no attribute 'drone'
'''},